src/OpenTelemetry.Exporter.Geneva/Metrics/Transport/MetricWindowsEventTracingDataTransport.cs

```diff
@@ -46,18 +41,12 @@ public unsafe void Send(MetricEventType eventType, byte[] data, int size)
 #endif
     public unsafe void SendOtlpProtobufEvent(byte[] data, int size)
     {
-        if (this.IsEnabled())
+        EventData descr = default;
+        fixed (byte* blob = &data[0])
```

Comment:
Wondering why this check had been removed here `if (data != null && data.Length != 0)`